docs/concept.md

```diff
@@ -14,12 +14,12 @@ Pinta uses layers to create a composite image. Think of layers as a stack of tra
 
 ![Stacked Layer](img/stacked.png)
 
-You can draw on each paper but still see the content of the other sheets through the transparent areas. Intermediate users often deal with images containing many layers. Layers do not need to be opaque and they do not cover the entire extent of an image. So when you look at an image's display, you may see more than just the top layer: you may see elements of many layers. 
+You can draw on each paper but still see the content of the other sheets through the transparent areas. Intermediate users often deal with images containing many layers. Layers do not need to be opaque and they do not cover the entirity of an image. So when you look at an image's display, you may see more than just the top layer: you may see elements of many layers. 
```

Comment:
I think you spelled **entirety** wrong....

Reply:
You're right 🤦 no more typing past 1 am haha

Reply:
haha no worries